core/types/src/attributes.rs

```diff
@@ -311,7 +324,7 @@ mod test {
         });
         assert_eq!(
             attributes.to_string(),
-            "Flags: (none) Xfrm: 0x7FFF_FFFF_FFFF_FFFF"
+            "Flags: (none) Xfrm: LEGACY | AVX | AVX_512 | MPX | PKRU | AMX | 0x7FFF_FFFF_FFF9_FD00"
```

Comment:
I think this output is better, now both the known flags as well as the left over bits are displayed.

Note that this slightly deviates from the test case above. That's because all the bits for the xfrm are part of the potential flags. While the flags member of Attributes does not allow for `0x10`  and a few other bits to be set.